ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/ui/bs/actions.coffee

```diff
@@ -241,7 +241,7 @@ angular.module('mc.core.ui.bs.actions', ['mc.util.ui.actions']).config ['actions
   ]
 
   actionsProvider.registerActionInRoles 'export', [actionsProvider.ROLE_LIST_ACTION, actionsProvider.ROLE_ITEM_ACTION, actionsProvider.ROLE_NAVIGATION, actionsProvider.ROLE_LIST_HEADER_ACTION], ['$scope', 'security', ($scope, security)->
-    return undefined unless security.hasRole('VIEWER')
+    return undefined unless security.hasRole('ADMIN') || security.hasRole('ROLE_METADATA_CURATOR')
```

Comment:
Now I see where these aggregated role comes from. I was wondering as I could not find them in backend. :sunglasses: